modules/gdscript/gdscript_vm.cpp

```diff
@@ -1813,7 +1814,7 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 				GET_INSTRUCTION_ARG(dst, argc);
 				*dst = Variant(); // Clear potential previous typed array.
 
-				*dst = Array(array, builtin_type, native_type, *script_type);
+				*dst = array;
```

Comment:
Should this (and the dict one) be `*dst = std::move(array)`? Not sure if the compiler is going to automatically convert this into a move assignment.

Reply:
Actually I'm not sure if that's even relevant, since `dst` is a `Variant`. So that's first going to have to do an implicit conversion to `Variant` and then do a move assignment from the rvalue. I wonder if the implicit conversion from local to `Variant` would benefit from `std::move`...